server/src/db/index.js

```diff
@@ -0,0 +1,23 @@
+const { MongoClient } = require('mongodb');
+
+const { appConfig } = require('../config/connectionStringBuilder');
```

Comment:
Does connectionStringBuilder export appConfig? I don't see it

Reply:
Connection determined in this file is no longer used and removed from repository.